src/eddymotion/utils.py

```diff
@@ -86,7 +84,7 @@ def bvalue_action(gradients):
 
     Parameters
     ----------
-    dwdata : :obj:`~eddymotion.dmri.DWI` DWI dataset, represented by this tool's internal type.
+    gradients : :obj:`numpy.ndarray` gradient table of the dataset
```

Comment:
This doesn't need the full table (b-vectors + b-values), here just the b-values should be used:

```suggestion
    bvalues : :obj:`list`
        List of b-values corresponding to all orientations of the dataset.
```

This makes me realize we want `size` to be also a `kwarg` because for this iterator we do not need the size (it is implicitly the same as the length of the b-values list).